requirements/base.txt

```diff
@@ -94,4 +94,5 @@ pandas
 wiki
 django-bulk-update
 elasticsearch
-
+Pygments==2.1.3
```

Comment:
CI failures 

`pkg_resources.VersionConflict: (Pygments 2.1.3 (/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages), Requirement.parse('Pygments>=2.2'))`

Could we update this and see if it still works as expected ?

Reply:
Alright.